internal/pkg/skuba/deployments/ssh/cri.go

```diff
@@ -17,11 +17,52 @@
 
 package ssh
 
+import (
+	"io/ioutil"
+	"path/filepath"
+
+	"github.com/SUSE/skuba/pkg/skuba"
+	"github.com/pkg/errors"
+	"k8s.io/klog"
+)
+
 func init() {
+	stateMap["cri.configure"] = criConfigure
 	stateMap["cri.start"] = criStart
+	stateMap["cri.reset"] = crioReset
+}
+
+func criConfigure(t *Target, data interface{}) error {
+	criFiles, err := ioutil.ReadDir(skuba.CriDir())
+	if err != nil {
+		return errors.Wrap(err, "Could not read local cri directory: "+skuba.CriDir())
+	}
+	defer t.ssh("rm -rf /tmp/cri.d")
+
+	for _, f := range criFiles {
+		if err := t.target.UploadFile(filepath.Join(skuba.CriDir(), f.Name()), filepath.Join("/tmp/cri.d", f.Name())); err != nil {
+			return err
+		}
+	}
+
+	_, _, err = t.ssh("mv /etc/sysconfig/crio /etc/sysconfig/crio.backup")
+	_, _, err = t.ssh("mv /tmp/cri.d/default_flags /etc/sysconfig/crio")
+	return err
 }
 
 func criStart(t *Target, data interface{}) error {
 	_, _, err := t.ssh("systemctl", "enable", "--now", "crio")
 	return err
 }
+
+func crioReset(t *Target, data interface{}) error {
+	_, _, err := t.ssh("test -f /etc/sysconfig/crio.backup")
+	if err == nil {
+		_, _, err = t.ssh("mv /etc/sysconfig/crio.backup /etc/sysconfig/crio")
+		if err != nil {
```

Comment:
I think this will fail if 1) `/etc/sysconfig/crio` exists and 2) you do not use `mv -f`

Besides that, maybe you could skip the `test` and just go ahead and ` t.ssh("mv -f /etc/sysconfig/crio.backup /etc/sysconfig/crio")` ignoring any errors...

Reply:
I added the `-f` but I didn't remove the error check. There's always something might go wrong  (e.g. `/etc/sysconfig/crio.backup` doesn't exist for some reason), so it doesn't harm.

Reply:
Well, it is a minor thing, but if you don't ignore the error, for exmaple a `reset` with no previous `node bootstrap/join` would lead to an error that could be ignorable... :confused:

Reply:
The `-f` would only be necessary if `mv` is an alias to something like `mv -i` or `mv -n`, which should never be the case in a non-interactive shell.

Either way, why not join those two commands using the shell, reducing the time window for a race condition and cutting an extra `ssh` out of the process?  `test -f $file && mv $file $dst`